src/app/static/templates/login.ftl

```diff
@@ -26,7 +26,11 @@
     </script>
 </head>
 <body>
-    <a href="https://www.unaids.org"><img src="public/images/unaids_logo.png" class="large-logo mx-auto mt-5"/></a>
+    <a href="https://www.unaids.org"><img src="public/images/unaids_logo.png" class="large-logo mx-auto mt-5 mb-4"/></a>
+    <#if RequestParameters?? && RequestParameters.redirectTo?? && RequestParameters.redirectTo?matches("explore")>
+        <h1 class="text-center"><strong>Naomi Data Exploration</strong></h1>
```

Comment:
```suggestion
        <h3 class="text-center"><strong>Naomi Data Exploration</strong></h3>
```

text is too large.

Reply:
I was going off the [mockups](https://imperiallondon.sharepoint.com/:p:/r/sites/MRCGIDA-hint-dev/_layouts/15/doc2.aspx?sourcedoc=%7BB91CFC01-B73F-44E3-9A30-D8E9BA47F710%7D&file=Naomi%20Data%20Exploration%20Mode%20mockups%20v2.pptx&action=edit&mobileredirect=true&cid=b66e037c-583b-4830-a171-999bcc5cd123&PreviousSessionID=f5d20f11-2094-0a0d-3d98-81bbe44f5c37) here